volkswagencarnet/vw_connection.py

```diff
@@ -479,17 +402,17 @@ async def _request(self, method, url, **kwargs):
 
             try:
                 if response.status == 204:
-                    res = {"status_code": response.status}
+                    res = {"status_code": response.status, "body": response.text}
```

Comment:
HTTP 204 has no response body. It will cause an error when a vehicle starts moving.
Previous code was correct: 
`res = {"status_code": response.status}`

Reply:
I really need to get my local repo straight... That change should have been removed again. Sorry.